### PR TITLE
Add exercises for anonymous functions

### PR DESCRIPTION
**Pull request by lalizita**
Add 3 exercises about anonymous functions in golang. Theses main.go files need to be fixed to run.

### CODE REVIEW

File: exercises/anonymous-functions/anonymouns-functions6/main.go [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+// Make me compile!
+
+// I AM NOT DONE
+package main
+
+import "fmt"
+
+func main() {
+
+	func(name string) {
```
Comment (mauricioabreu) [inline exercises/anonymous-functions/anonymouns-functions6/main.go:11]: I could not understand the relation of the function with the hint you provided. Maybe you forgot to push the code?

The exercise name could be fixed: `anonymous-functions3`

Shouldn't it be `anonymous`? I think that is a typo

**Review by mauricioabreu — CHANGES_REQUESTED** [review]
I'm glad you are contributing to `golings`

Do you mind fixing the problems I have pointed out?

Thank you!

File: exercises/anonymous-functions/anonymouns-functions2/main.go [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+func main() {
+
+	var sayBye func(name string)
+
+	sayBye = func() {
+		fmt.Printf("Bye %s", n)
+	}
+
+	sayBye(123)
```
[the remark this answers is not in the document]
Comment (lalizita) [reply exercises/anonymous-functions/anonymouns-functions2/main.go]: Yeah! You're right!

File: exercises/anonymous-functions/anonymouns-functions1/main.go [hunk 3]
```diff
@@ -0,0 +1,32 @@
+// anonymous functions3
+// Make me compile!
+
+// I AM NOT DONE
+package main
+
+import "fmt"
+
+func status() func() string {
```
[the remark this answers is not in the document]
Comment (lalizita) [reply exercises/anonymous-functions/anonymouns-functions1/main.go:9]: Thank you!

File: exercises/anonymous-functions/anonymouns-functions6/main.go (hunk 1 above)
Comment (lalizita) [reply exercises/anonymous-functions/anonymouns-functions6/main.go]: Sorry, I did a mistake, forgot to rename some comments and folders. I'm gonna fix it and improve the anonymous-functions2 hint

**Review by mauricioabreu — APPROVED** [review]

**Comment by mauricioabreu** [comment]
Thank you for your contribution! 🥳

**Pull request merged**